src/aspire/utils/filters.py

```diff
@@ -136,6 +136,7 @@ def _evaluate(self, omega):
 
     def scale(self, c):
         self._scale *= c
+        return self
```

Comment:
It's modifying itself, but also returning itself so that these calls can be chained:

```
f.scale(2).evaluate(omegas)
```

 `return self` is common to support such scenarios.